Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,16 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const durationInHours = tasks.map(minute => minute.duration / 60);
+  const moreThan2Hours = durationInHours.filter(newArray => newArray >= 2);
```

Comment:
Since we are filtering the array `durationsInHours` a single element is a `durationInHours`, not a `newArray`. Therefore this line would be more accurate as it read as:

```js
const moreThan2Hours = durationsInHours.filter(durationInHours => durationInHours >= 2);
```

The variable name `moreThan2Hours` is incorrect, because we also include durations that are equal to two. Therefore you should rename this to a more accurate name, e.g.: `durationsEqualOrMoreThan2Hours` os something similar.